frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -324,17 +324,21 @@ const allHostTableHeaders: IDataColumn[] = [
     Cell: (cellProps: ICellProps) => <TextCell value={cellProps.cell.value} />,
   },
   {
-    title: "Uptime",
+    title: "Last restarted",
     Header: (cellProps: IHeaderProps) => (
       <HeaderCell
         value={cellProps.column.title}
         isSortedDesc={cellProps.column.isSortedDesc}
       />
     ),
     accessor: "uptime",
-    Cell: (cellProps: ICellProps) => (
-      <TextCell value={cellProps.cell.value} formatter={humanHostUptime} />
```

Comment:
Do we not use the formatter prop anymore on TextCells? If not thats fine, but if we often other places I think we use I think we should use that prop here.

Reply:
The formatter only takes 1 string/number/boolean value, this cell requires two values, should we refactor the formatter to allow for multiple?

Reply:
ah ok i see. hum in this case i think its ok to leave as is but maybe leave a comment why we format differently here.